mlrun/kfpops.py

```diff
@@ -153,7 +154,7 @@ def mlrun_op(
     mode: str = "",
     handler: str = "",
     more_args: list = None,
-    tuning_strategy=None,
```

Comment:
Why removing ? this break BC

Reply:
wasn't used by the caller anyway (`base.as_step()`)